packages/engine/Source/Scene/VoxelProvider.js

```diff
@@ -214,17 +203,17 @@ Object.defineProperties(VoxelProvider.prototype, {
 });
 
 /**
- * Requests the data for a given tile. The data is a flattened 3D array ordered by X, then Y, then Z.
- * This function should not be called before {@link VoxelProvider#ready} returns true.
- * @function
+ * Requests the data for a given tile.
+ *
+ * @private
```

Comment:
Since this is a public interface which defines a "contract", I'm curious why this method should be private. For instance, `TerrainProvider.prototype.requestTileGeometry` is not private. 

I think it makes sense to say in the docs that this function should not be called directly, but I think making it explicitly private goes against the purpose of an interface.

Reply:
This was based on making things more like `Cesium3DTileset`, which manages tile requests via private methods like `Cesium3DTileset.prototype.update`.
If we want to continue supporting custom providers, then I guess leaving this method public could make the documentation better for people trying to build a provider.